terasoluna-gfw-common-libraries/terasoluna-gfw-validator/src/test/java/org/terasoluna/gfw/common/validator/constraintvalidators/ConstraintValidatorsUtilsTest.java

```diff
@@ -0,0 +1,78 @@
+package org.terasoluna.gfw.common.validator.constraintvalidators;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+
+import java.lang.reflect.Constructor;
+
+import javax.validation.ConstraintValidatorContext;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class ConstraintValidatorsUtilsTest {
+
+    private CompareValidator compareValidator;
+
+    @Before
+    public void setUp() {
+        compareValidator = new CompareValidator();
+    }
+
+    @Test
+    public void testConstraintValidatorsUtils() throws Exception {
+        // set up
+        Constructor<ConstraintValidatorsUtils> constructor = ConstraintValidatorsUtils.class.getDeclaredConstructor();
+        assertThat(constructor.isAccessible(), is(false));
+        constructor.setAccessible(true);
+
+        // assert
+        assertNotNull(constructor.newInstance());
+
+        constructor.setAccessible(false);
+    }
+
+    @Test
+    public void testGetPropertyValueBeanAndPropertyNameNotNull() throws Exception {
+        // set up
+        Object bean = mock(Object.class);
+        ConstraintValidatorContext context = mock(ConstraintValidatorContext.class);
+
+        // assert
+        assertTrue(compareValidator.isValid(bean, context));
```

Comment:
- [ ] I think these test cases art too fat.

It seems to be enough with follows:
```java
Object o = ConstraintValidatorsUtils.getPropertyValue(any object, any property name);
assertThat(o, notNullValue);
```